.replit

```diff
@@ -1,2 +1,10 @@
+run = "npm start"
+entrypoint = "server.js"
+
+[packager]
 language = "nodejs"
```

Comment:
What is the point of having this, if we still need to add that line to `.gitattributes`?

Reply:
https://docs.replit.com/programming-ide/configuring-repl#packager

> [packager]
> Description: Package management configuration. Learn more about installing packages [here](https://docs.replit.com/repls/packages/#DirectImports).

That config setting doesn't seem to affect the language auto-detection Replit does when importing from GitHub. As long as Linguist detects some language as the one with the most code (i.e. the repo language statistics) Replit seems to use that as the information about which language it should auto-detect.